src/libraries/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/HKDF.cs

```diff
@@ -110,10 +115,15 @@ public static byte[] Expand(HashAlgorithmName hashAlgorithmName, byte[] prk, int
         /// <param name="prk">The pseudorandom key of at least <see cref="HashLength"/> bytes (usually the output from Expand step).</param>
         /// <param name="output">The destination buffer to receive the output keying material.</param>
         /// <param name="info">The context and application specific information (can be an empty span).</param>
+        /// <exception cref="ArgumentException"><paramref name="output"/> is larger than the maximum allowed length.</exception>
```

Comment:
It seems like this should be a single entry that throws this when either case is true, not be two separate throws elements.

Reply:
Oh, good point.  And they're the same parameter, so easy to squish together.

Reply:
Good point, they are now combined.